lib/pci.py

```diff
@@ -403,23 +540,39 @@ def get_pci_name(pci_address):
         return " ".join(pci_name)
     return ""
 
-
-def get_driver(pci_address):
+def get_driver(adapter_type, pci_address):
```

Comment:
keep adapter_type optional, ie  as second param and the legacy code would work only for pci_address

Reply:
get_driver function will handle both Network and Block devices.    nvmf is the  one off case which has multiple drivers to be accounted.

Reply:
yes, agree with Abdul. may have  `adapter_type` has `None` as default

Reply:
As discussed with Abdul offline.  this is not required. marking it as resolved.